Can't build docs locally

### Setup Information

- Xclim version: master
- Python version: 3.10
- Operating System: Fedora 36


### Description

When I try to build the documentation the official way, it fails.


### Steps To Reproduce

In the root directory of  `xclim`: `make docs`
The "apidoc" step passes, as does the "linkcheck" one (with 2 warnings), but the "html" fails with:
```
make -C docs html
make[1] : on entre dans le répertoire « /home/pbourg/Projets/xclim/docs »
Sphinx v5.3.0 en cours d'exécution
Création du répertoire de sortie... fait
Chargement de l'environnement pickled... checking bibtex cache... up to date
fait
Construction en cours [mo] : cibles périmées pour les fichiers po 0
Construction [html] : cibles périmées pour les fichiers sources 40
Mise à jour de l'environnement : 0 ajouté(s), 0 modifié(s), 0 supprimé(s)
Recherche des fichiers périmés... aucun résultat
Document en préparation... fait
/home/pbourg/Projets/xclim/HISTORY.rst:299: WARNING: type de node inconnu : <SkipMarker: >                                                                                         

Une exception a été levée :
  File "/home/pbourg/mambaforge/envs/xclim/lib/python3.10/site-packages/docutils/nodes.py", line 2056, in unknown_departure
    raise NotImplementedError(
NotImplementedError: <class 'sphinx.writers.html5.HTML5Translator'> departing unknown node type: SkipMarker
La trace d’appels complète a été sauvegardée dans /tmp/sphinx-err-36pdrc5f.log, au cas où vous souhaiteriez signaler le problème aux développeurs.
Merci de rapporter ceci s'il s'agit d'une erreur utilisateur, afin d'améliorer le message d'erreur à l'avenir.
Un rapport d'erreur peut être déposé dans le système de tickets à <https://github.com/sphinx-doc/sphinx/issues>. Merci !
make[1]: *** [Makefile:20 : html] Erreur 2
make[1] : on quitte le répertoire « /home/pbourg/Projets/xclim/docs »
make: *** [Makefile:88 : docs] Erreur 2
```

### Additional context

I seem to have every package up to date.

### Contribution

- [X] I would be willing/able to open a Pull Request to address this bug.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -98,6 +98,10 @@ jobs:
           channels: conda-forge,defaults
           mamba-version: "*"
           python-version: "3.9"
+      - name: Conda and Mamba versions
+        shell: bash -l {0}
+        run: |
+          mamba --version
       - name: Install tox
         shell: bash -l {0}
         run: |

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -101,6 +101,7 @@ jobs:
       - name: Conda and Mamba versions
         shell: bash -l {0}
         run: |
+          mamba update -n base --all
           mamba --version
       - name: Install tox
         shell: bash -l {0}

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -92,7 +92,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Setup conda
-        uses: conda-incubator/setup-miniconda@v2.1.1
+        uses: conda-incubator/setup-miniconda@v2.2.0
         with:
           activate-environment: xclim39
           channels: conda-forge,defaults

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -92,29 +92,46 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Setup conda
-        uses: conda-incubator/setup-miniconda@v2.2.0
+        uses: conda-incubator/setup-miniconda@v2
         with:
+          auto-update-conda: true
           activate-environment: xclim39
           channels: conda-forge,defaults
+          environment-file: environment.yml
           mamba-version: "*"
           python-version: "3.9"
       - name: Conda and Mamba versions
         shell: bash -l {0}
         run: |
-          mamba update -n base --all
           mamba --version
-      - name: Install tox
+      - name: Install dependencies
         shell: bash -l {0}
         run: |
-          mamba install -n xclim39 tox-conda
-      - name: Test with tox
+          mamba install -n xclim39 eofs eigen pybind11
+          conda activate xclim39
+          python -m pip install --no-user "git+https://github.com/Ouranosinc/SBCK.git@easier-install#egg=sbck&subdirectory=python"
+          python -m pip install --no-user .
+      - name: Test
         shell: bash -l {0}
         run: |
           conda activate xclim39
-          tox -e opt-slow-mamba
+          pip check
+          pytest xclim --durations=10
         env:
           CONDA_EXE: mamba
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#      - name: Install tox
+#        shell: bash -l {0}
+#        run: |
+#          mamba install -n xclim39 tox-conda
+#      - name: Test
+#        shell: bash -l {0}
+#        run: |
+#          conda activate xclim39
+#          tox -e opt-slow-mamba
+#        env:
+#          CONDA_EXE: mamba
+#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Report coverage
         uses: AndreMiras/coveralls-python-action@develop
         with:

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -3,12 +3,12 @@ default_language_version:
 
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.2.0
+    rev: v3.2.3
     hooks:
       - id: pyupgrade
         args: [ "--py38-plus" ]
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
     hooks:
       - id: trailing-whitespace
         exclude: setup.cfg
@@ -21,7 +21,7 @@ repos:
       - id: pretty-format-json
         args: ['--autofix', '--no-ensure-ascii', '--no-sort-keys']
   - repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
+    rev: 6.0.0
     hooks:
       - id: flake8
         additional_dependencies: [ 'flake8-rst-docstrings' ]

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -27,6 +27,7 @@ Breaking changes
 Bug fixes
 ^^^^^^^^^
 * The weighted ensemble statistics are now performed within a context in order to preserve data attributes. (:issue:`1232`, :pull:`1234`).
+* The `make docs` Makefile recipe was failing with an esoteric error. This has been resolved by splitting the `linkcheck` and `docs` steps into separate actions. (:issue:`1248`. :pull:`1251`).
 
 Internal changes
 ^^^^^^^^^^^^^^^^

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -33,6 +33,8 @@ Internal changes
 * Minor adjustments to GitHub Actions workflows (newest Ubuntu images, updated actions version, better CI triggering). (:pull:`1221`).
 * Updated article from Alavoine & Grenier (2022) within documentation. Many article reference URLs have been updated to use HTTPS where possible. (:issue:`1246`, :pull:`1247`).
 * Added relevant variable dataflag checks for potential evaporation, convective precipitation, and air pressure at sea level. (:pull:`1241`).
+* Documentation restructured to include `ReadMe` page (as `About`) with some minor changes to documentation titles. (:pull:`1233`).
+* `xclim` development build now uses `nbqa` to effectively run black checks over notebook cells. (:pull:`1233`).
 
 0.39.0 (2022-11-02)
 -------------------

--- a/Makefile
+++ b/Makefile
@@ -84,7 +84,7 @@ autodoc: clean-docs ## create sphinx-apidoc files:
 linkcheck: autodoc ## run checks over all external links found throughout the documentation
 	$(MAKE) -C docs linkcheck
 
-docs: linkcheck ## generate Sphinx HTML documentation, including API docs
+docs: autodoc ## generate Sphinx HTML documentation, including API docs
 	$(MAKE) -C docs html
 ifndef READTHEDOCS
 	$(BROWSER) docs/_build/html/index.html

--- a/Makefile
+++ b/Makefile
@@ -59,7 +59,7 @@ lint: ## check style with flake8 and black
 	pydocstyle --config=setup.cfg xclim
 	flake8 --config=setup.cfg xclim
 	black --check --target-version py38 xclim
-	black --check --ipynb --target-version py38 docs --include "\.ipynb$$"
+	nbqa black --check docs --target-version py38
 	blackdoc --check --target-version py38 xclim --exclude xclim/indices/__init__.py,xclim/docs/installation.rst
 	isort --check --settings-file=setup.cfg xclim --add_imports="from __future__ import annotations"
 	pylint --rcfile=pylintrc --exit-zero xclim

--- a/README.rst
+++ b/README.rst
@@ -1,12 +1,16 @@
-==========================================
+======================================
 xclim: Climate services library |logo|
-==========================================
+======================================
 
-+----------------+----------------------------+-----------------------------------+-----------------------------------------------------+
-| Downloads      | Documentation and Support  | Open Source                       | Development Status                                  |
-+----------------+----------------------------+-----------------------------------+-----------------------------------------------------+
-| |pypi| |conda| | |docs| |gitter|            | |license| |fair| |fossa| |zenodo| | |build| |pre-commit| |coveralls| |black| |security| |
-+----------------+----------------------------+-----------------------------------+-----------------------------------------------------+
++----------------------------+-----------------------------------------------------+
+| Downloads                  | |pypi| |conda|                                      |
++----------------------------+-----------------------------------------------------+
+| Documentation and Support  | |docs| |gitter|                                     |
++----------------------------+-----------------------------------------------------+
+| Open Source                | |license| |fair| |fossa| |zenodo|                   |
++----------------------------+-----------------------------------------------------+
+| Development Status         | |build| |pre-commit| |coveralls| |black| |security| |
++----------------------------+-----------------------------------------------------+
 
 
 `xclim` is an operational Python library for climate services, providing numerous climate-related indicator tools
@@ -19,6 +23,7 @@ Leveraging xarray and dask, users can easily bias-adjust climate simulations ove
 
 For example, the following would compute monthly mean temperature from daily mean temperature:
 
+.. autolink-skip::
 .. code-block:: python
 
     import xclim
@@ -37,11 +42,11 @@ streamflow and sea ice concentration, numerous bias-adjustment algorithms, as we
 .. _dask: https://docs.dask.org/
 
 Documentation
--------------
+^^^^^^^^^^^^^
 The official documentation is at https://xclim.readthedocs.io/
 
-Contributing
-------------
+Contributing to xclim
+^^^^^^^^^^^^^^^^^^^^^
 xclim is in active development and is being used in production by climate services specialists around the world.
 
 * If you're interested in participating in the development of xclim by suggesting new features, new indices or report bugs, please leave us a message on the `issue tracker`_. There is also a chat room on gitter (|gitter|).
@@ -52,21 +57,20 @@ xclim is in active development and is being used in production by climate servic
 .. _Contributing Guidelines: https://github.com/Ouranosinc/xclim/blob/master/.github/CONTRIBUTING.rst
 
 How to cite this library
-------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^
 If you wish to cite xclim in a research publication, we kindly ask that you use the bibliographical reference information available through `Zenodo`_
 
 .. _Zenodo: https://doi.org/10.5281/zenodo.2795043
 
 Credits
--------
+^^^^^^^
+xclim development is funded through Ouranos_, Environment and Climate Change Canada (ECCC_), the `Fonds vert`_ and the Fonds d'électrification et de changements climatiques (FECC_), the Canadian Foundation for Innovation (CFI_), and the Fonds de recherche du Québec (FRQ_).
 
-xclim development is funded through Ouranos_, Environment and Climate Change Canada (ECCC_), the `Fonds vert`_ and the Fonds d’électrification et de changements climatiques (FECC_), the Canadian Foundation for Innovation (CFI_), and the Fonds de recherche du Québec (FRQ_).
+This package was created with Cookiecutter_ and the `audreyfeldroy/cookiecutter-pypackage`_ project template.
 
-This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypackage`_ project template.
-
-.. _audreyr/cookiecutter-pypackage: https://github.com/audreyr/cookiecutter-pypackage/
+.. _audreyfeldroy/cookiecutter-pypackage: https://github.com/audreyfeldroy/cookiecutter-pypackage/
 .. _CFI: https://www.innovation.ca/
-.. _Cookiecutter: https://github.com/audreyr/cookiecutter/
+.. _Cookiecutter: https://github.com/cookiecutter/cookiecutter/
 .. _ECCC: https://www.canada.ca/en/environment-climate-change.html
 .. _FECC: https://www.environnement.gouv.qc.ca/ministere/fonds-electrification-changements-climatiques/index.htm
 .. _Fonds vert: https://www.environnement.gouv.qc.ca/ministere/fonds-vert/index.htm
@@ -118,7 +122,7 @@ This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypack
         :alt: FOSSA
 
 .. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
-        :target: https://github.com/python/black
+        :target: https://github.com/psf/black
         :alt: Python Black
 
 .. |logo| image:: https://raw.githubusercontent.com/Ouranosinc/xclim/master/_static/_images/xclim-logo-small.png

--- a/docs/_static/style.css
+++ b/docs/_static/style.css
@@ -58,3 +58,10 @@ dt var {
 ul.simple li {
  list-style: circle;
 }
+
+/*this selects the contents section, the first table of contents and the list item*/
+/*underneath it. Then it removes the spacing to make it look whole. If you have more*/
+/*items, you can consider using :not(:last-of-type) selector.*/
+#contents .toctree-wrapper:first-of-type ul {
+    margin-bottom: 0;
+}

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -115,9 +115,11 @@ autosectionlabel_prefix_document = True
 autosectionlabel_maxdepth = 2
 
 linkcheck_ignore = [
-    r"https://github.com/Ouranosinc/xclim/(pull|issue).*",
+    r"https://github.com/Ouranosinc/xclim/(pull|issue).*",  # too labourious to fully check
     r"https://doi.org/10.1093/mnras/225.1.155",  # does not allow linkcheck requests (error 403)
+    r"https://www.ouranos.ca",  # bad ssl certificate
 ]
+linkcheck_exclude_documents = [r"readme"]
 
 napoleon_numpy_docstring = True
 napoleon_use_rtype = False
@@ -322,3 +324,7 @@ texinfo_documents = [
         "Miscellaneous",
     )
 ]
+
+
+def setup(app):
+    app.add_css_file("_static/style.css")

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -58,26 +58,26 @@ def _indicator_table(module):
     """Return a sequence of dicts storing metadata about all available indices in xclim."""
     inds = _get_indicators(getattr(xclim.indicators, module))
     table = {}
-    for indname, ind in inds.items():
+    for ind_name, ind in inds.items():
         # Apply default values
         # args = {
         #     name: p.default if p.default != inspect._empty else f"<{name}>"
         #     for (name, p) in ind._sig.parameters.items()
         # }
         try:
-            table[indname] = ind.json()  # args?
+            table[ind_name] = ind.json()  # args?
         except KeyError as err:
             warnings.warn(
                 f"{ind.identifier} could not be documented.({err})", UserWarning
             )
         else:
-            table[indname]["doc"] = ind.__doc__
+            table[ind_name]["doc"] = ind.__doc__
             if ind.compute.__module__.endswith("generic"):
-                table[indname][
+                table[ind_name][
                     "function"
                 ] = f"xclim.indices.generic.{ind.compute.__name__}"
             else:
-                table[indname]["function"] = f"xclim.indices.{ind.compute.__name__}"
+                table[ind_name]["function"] = f"xclim.indices.{ind.compute.__name__}"
     return table
 
 
@@ -117,7 +117,9 @@ autosectionlabel_maxdepth = 2
 linkcheck_ignore = [
     r"https://github.com/Ouranosinc/xclim/(pull|issue).*",  # too labourious to fully check
     r"https://doi.org/10.1093/mnras/225.1.155",  # does not allow linkcheck requests (error 403)
-    r"https://www.ouranos.ca",  # bad ssl certificate
+    r"https://www.ouranos.ca/.*",  # bad ssl certificate
+    r"https://doi.org/10.1080/.*",  # tandfonline does not allow linkcheck requests (error 403)
+    r"https://www.tandfonline.com/.*",  # tandfonline does not allow linkcheck requests (error 403)
 ]
 linkcheck_exclude_documents = [r"readme"]
 

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -1,19 +1,16 @@
 xclim Official Documentation
 ============================
 
-``xclim`` is a library of functions to compute climate indices from observations or model simulations. It is built using `xarray`_ and can benefit from the parallelization handling provided by `dask`_. Its objective is to make it as simple as possible for users to compute indices from large climate datasets and for scientists to write new indices with very little boilerplate.
+.. toctree::
+   :maxdepth: 0
+   :caption: Table of Contents:
 
-For applications where meta-data and missing values are important to get right, xclim provides a class for each index that validates inputs, checks for missing values, converts units and assigns metadata attributes to the output. This also provides a mechanism for users to customize the indices to their own specifications and preferences.
-
-xclim currently provides over 50 indices related to mean, minimum and maximum daily temperature, daily precipitation, streamflow and sea ice concentration.
-
-.. _xarray: https://docs.xarray.dev/
-.. _dask: https://www.dask.org/
+   self
 
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
 
+   About <readme>
    installation
    notebooks/usage
    notebooks/index

--- a/docs/indicators.rst
+++ b/docs/indicators.rst
@@ -1,5 +1,5 @@
 ==================
-Climate indicators
+Climate Indicators
 ==================
 
 :py:class:`xclim.core.indicator.Indicator` instances essentially perform the same computations as the functions

--- a/docs/indices.rst
+++ b/docs/indices.rst
@@ -1,5 +1,5 @@
 ===============
-Climate indices
+Climate Indices
 ===============
 
 .. automodule:: xclim.indices

--- a/docs/notebooks/cli.ipynb
+++ b/docs/notebooks/cli.ipynb
@@ -4,8 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Command line interface\n",
-    "======================\n",
+    "# Command Line Interface\n",
     "\n",
     "xclim provides the  `xclim` command line executable to perform basic indicator\n",
     "computation easily without having to start up a full python environment. However, not\n",
@@ -162,7 +161,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this example, we decided to use `tasmin` for the `tas` variable. We didn't need to provide the `--pr` parameter as our data has the same name. \n",
+    "In this example, we decided to use `tasmin` for the `tas` variable. We didn't need to provide the `--pr` parameter as our data has the same name.\n",
     "\n",
     "Finally, more than one indicators can be computed to the output dataset by simply chaining the calls:"
    ]
@@ -224,9 +223,9 @@
     "Data Quality Checks\n",
     "-------------------\n",
     "\n",
-    "As of version 0.30.0, `xclim` now also provides a command-line utility for performing data quality control checks on existing NetCDF files. \n",
+    "As of version 0.30.0, `xclim` now also provides a command-line utility for performing data quality control checks on existing NetCDF files.\n",
     "\n",
-    "These checks examine the values of data_variables for suspicious value patterns (e.g. values that repeat for many days) or erroneous values (e.g. humidity percentages outside of 0-100, minimum temperatures exceeding maximum temperatures, etc.). The checks (called \"data flags\") are based on the ECAD ICCLIM quality control checks (https://www.ecad.eu/documents/atbd.pdf). \n",
+    "These checks examine the values of data_variables for suspicious value patterns (e.g. values that repeat for many days) or erroneous values (e.g. humidity percentages outside of 0-100, minimum temperatures exceeding maximum temperatures, etc.). The checks (called \"data flags\") are based on the ECAD ICCLIM quality control checks (https://www.ecad.eu/documents/atbd.pdf).\n",
     "\n",
     "The full list of checks performed for each variable are listed in `xclim/core/data/variables.yml`."
    ]
@@ -376,7 +375,7 @@
     "ds3 = xr.open_dataset(\"flag_output_pr.nc\")\n",
     "display(ds3.data_vars)\n",
     "for dv in ds3.data_vars:\n",
-    "    display(ds3[dv])"
+    "    display(ds3[dv])\n"
    ]
   }
  ],

--- a/docs/notebooks/units.ipynb
+++ b/docs/notebooks/units.ipynb
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Unit handling"
+    "# Unit Handling"
    ]
   },
   {
@@ -152,7 +152,7 @@
    "source": [
     "## Other utilites\n",
     "\n",
-    "Many helper functions are defined in `xclim.core.units`, see [Unit handling module](../api.rst#unit-handling-module)."
+    "Many helper functions are defined in `xclim.core.units`, see [Unit handling module](../api.rst#unit-handling-module).\n"
    ]
   }
  ],

--- a/docs/readme.rst
+++ b/docs/readme.rst
@@ -1,0 +1,1 @@
+.. include:: ../README.rst

--- a/docs/sdba.rst
+++ b/docs/sdba.rst
@@ -1,5 +1,5 @@
 ==========================================
-Bias adjustment and downscaling algorithms
+Bias Adjustment and Downscaling Algorithms
 ==========================================
 
 `xarray` data structures allow for relatively straightforward implementations of simple bias-adjustment and downscaling algorithms documented in :ref:`Adjustment Methods <sdba:SDBA User API>`.

--- a/requirements_dev.txt
+++ b/requirements_dev.txt
@@ -7,6 +7,7 @@ flake8
 flake8-rst-docstrings
 ipython
 mypy
+nbqa
 nbval
 pre-commit>=2.9
 pydocstyle>=5.1.1

--- a/tox.ini
+++ b/tox.ini
@@ -21,13 +21,14 @@ deps =
     black[jupyter]
     blackdoc
     isort
+    nbqa
     pydocstyle
     yamllint
 commands =
     pydocstyle --config=setup.cfg xclim
     flake8 xclim
     black --check --target-version py38 xclim
-    black --check --ipynb --target-version py38 docs --include "\.ipynb$"
+    nbqa black --check docs --target-version py38
     blackdoc --check --target-version py38 xclim --exclude xclim/indices/__init__.py
     isort --check --settings-file=setup.cfg xclim
     yamllint --config-file .yamllint.yaml xclim

--- a/xclim/indices/run_length.py
+++ b/xclim/indices/run_length.py
@@ -1,6 +1,5 @@
 # noqa: D205,D400
 """
-# TODO: Remove default values from
 Run length algorithms submodule
 ===============================
 
@@ -48,6 +47,8 @@ def use_ufunc(
         Input array.
     dim : str
         The dimension along which to find runs.
+    freq : str
+      Resampling frequency.
     index : {'first', 'last'}
         If 'first' (default), the run length is indexed with the first element in the run.
         If 'last', with the last element in the run.
@@ -93,17 +94,17 @@ def resample_and_rl(
       N-dimensional array (boolean).
     resample_before_rl : bool
       Determines whether if input arrays of runs `da` should be separated in period before
-      or after the run length algorithms are applied
+      or after the run length algorithms are applied.
     compute
       Run length function to apply
     args
-      Positional arguments needed in `compute`
+      Positional arguments needed in `compute`.
     dim: str
       The dimension along which to find runs.
     freq : str
       Resampling frequency.
     kwargs
-      Keyword arguments needed in `compute`
+      Keyword arguments needed in `compute`.
 
     Returns
     -------
@@ -997,7 +998,7 @@ def first_run_1d(arr: Sequence[int | float], window: int) -> int | np.nan:
         Returns np.nan if there are no valid runs.
     """
     v, rl, pos = rle_1d(arr)
-    ind = np.where(v * rl >= window, pos, np.inf).min()
+    ind = np.where(v * rl >= window, pos, np.inf).min()  # noqa
 
     if np.isinf(ind):
         return np.nan

--- a/xclim/testing/tests/test_sdba/test_properties.py
+++ b/xclim/testing/tests/test_sdba/test_properties.py
@@ -376,6 +376,7 @@ def test_spatial_correlogram():
     )
 
 
+@pytest.skip("Hangs inexplicably.")
 def test_first_eof():
     pytest.importorskip("eofs")
     sim = open_dataset("NRCANdaily/nrcan_canada_daily_tasmax_1990.nc").tasmax.isel(

--- a/xclim/testing/tests/test_sdba/test_properties.py
+++ b/xclim/testing/tests/test_sdba/test_properties.py
@@ -376,7 +376,7 @@ def test_spatial_correlogram():
     )
 
 
-@pytest.mark.skip("Hangs inexplicably.")
+@pytest.mark.skip(reason="Hangs inexplicably.")
 def test_first_eof():
     pytest.importorskip("eofs")
     sim = open_dataset("NRCANdaily/nrcan_canada_daily_tasmax_1990.nc").tasmax.isel(

--- a/xclim/testing/tests/test_sdba/test_properties.py
+++ b/xclim/testing/tests/test_sdba/test_properties.py
@@ -376,7 +376,7 @@ def test_spatial_correlogram():
     )
 
 
-@pytest.skip("Hangs inexplicably.")
+@pytest.mark.skip("Hangs inexplicably.")
 def test_first_eof():
     pytest.importorskip("eofs")
     sim = open_dataset("NRCANdaily/nrcan_canada_daily_tasmax_1990.nc").tasmax.isel(

